docs/dive-in/2024/01/2024-01-24-github-actions/2024-01-24-github-actions.md

```diff
@@ -0,0 +1,29 @@
+# TITLE: Set up our first Github Action
+
+Author : Barnabé PILLIAUDIN
+Date : 2024-01-24
+
+## SUMMARY:
+
+We wan to have our code linted and tested automatically when pushed. As we our code is hosted on Github, we will use Github Actions to do so.
+
+## CONTEXT:
+
+Having code linted and tested is a good practice and will help us maintaining a clean and working codebase. We it to be done automatically when code is push to be sure that the code merged into staging to be deployed is working and clean. It will also help the reviewers.
```

Comment:
```suggestion
Having code linted and tested is a good practice and will help us maintain a clean and working codebase. It would be done automatically when the code is pushed to be sure that the code merging into staging will work and be clean. It will also help the reviewers.
```